### PR TITLE
Integrate video rendering logics (Jupyter & Colab)

### PR DESCRIPTION
**Pull request by Curt-Park**
## Done
- Fixed packages for Apple M1 chip.
- Integration of video rendering logics for Jupyter and Colab.
- `gym.wrapper.Monitor` is replaced with `gym.wrapper.RecordVideo`. (`gym.wrapper.Monitor` will be deprecated from gym)

## Colab
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/01.dqn.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/02.double_q.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/03.per.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/04.dueling.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/05.noisy_net.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/06.categorical_dqn.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/07.n_step_learning.ipynb
- https://colab.research.google.com/github/Curt-Park/rainbow-is-all-you-need/blob/feature%2Frendering-integration/08.rainbow.ipynb

### CODE REVIEW

**Comment by Curt-Park** [comment]
@MrSyee Please check that the window is properly closed after agent.test() is done.

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,5 @@
-numpy==1.20.0
-torch==1.7.1
-matplotlib==3.0.2
-JSAnimation==0.1
-pyglet==1.5.11
-gym
-typing
-jupyter
+pytorch==1.9.0
```
Comment (MrSyee) [inline requirements.txt:1]: should fix `pytorch` -> `torch`

**Comment by MrSyee** [comment]
> @MrSyee Please check that the window is properly closed after agent.test() is done.

I've checked that the window is properly closed.

**Comment by Curt-Park** [comment]
@MrSyee thanks for your comment. pytorch is a proper expression for the conda's requirement. I replaced pip install with conda install for m1 chip compatibility. Please see the updated Readme.

**Comment by MrSyee** [comment]
> @MrSyee thanks for your comment. pytorch is a proper expression for the conda's requirement. I replaced pip install with conda install for m1 chip compatibility. Please see the updated Readme.

I got it.

**Pull request merged**